README.md

```diff
@@ -1,9 +1,21 @@
 # Eludris
 
-A simple FOSS federated decentralized blazingly fast(:rocket:) social media
-platform with e2ee support made in rust(:rocket:). :rocket::rocket::rocket::rocket::rocket::rocket::rocket::rocket::rocket::rocket::rocket:
+A free and open source, federated, End-to-end-Encrypted social media platform made in rust that's easy to deploy and configure while striving to be *truly **yours***.
+
+## Deployment
+
+We really recommend and *only* officially support using the provided docker-compose as a quick way to get stuff running, just edit your `Eludris.toml` to suit your needs then run
+
+```sh
+docker-compose up
+```
+
+Congratulations, you've now successfully deployed your Eludris instance!
 
 ## Default Ports
 
 Oprish: 7159
```

Comment:
As a new user has no idea what these are when looking at the README alone, perhaps add links to these names so they know one is http, another is ws and another is a cdn

Reply:
Great idea, perhaps now is the time to start MDBook based Eludris docs which aim to explain everything Eludris related + API docs?